lib/Switch.js

```diff
@@ -47,8 +47,8 @@ export class Switch extends Component {
     renderInsideCircle: () => null,
     innerCircleStyle: {},
     disabled: false,
-    activeText: "On",
-    inActiveText: "Off",
+    activeText: "Yes",
```

Comment:
Please don't change default props. We should keep legacy support for all developers/projects.